src/main/java/org/apache/commons/lang3/RandomStringUtils.java

```diff
@@ -448,6 +458,7 @@ public static String random(int count, int start, int end, final boolean letters
      * @return the random string
      * @throws IllegalArgumentException if {@code count} &lt; 0 or the string is empty.
      */
+    @SuppressWarnings("index:argument.type.incompatible") // null array is handled differently
```

Comment:
Why does the `null` array being handled differently mean that a warning is issued? Shouldn't the spec be accurate regardless?